tests/unit/test_chirp_common.py

```diff
@@ -715,7 +715,9 @@ class TestOverrideRules(base.BaseTest):
     # please ask permission first.
     IMMUTABLE_WHITELIST = [
         # Uncomment me when the time comes
+        'BTECH_GMRS-20V2',
         'BTECH_GMRS-V2',
+        'Radioddity_DB25-G',
```

Comment:
Is this the error?
```

    def test_radio_overrides(self):
        for rclass in directory.DRV_TO_RADIO.values():
            if directory.radio_class_id(rclass) in self.IMMUTABLE_WHITELIST:
                self._test_radio_override_calls_super(rclass)
            else:
>               self._test_radio_override_immutable_policy(rclass)
```

I thought it was different. This is because one of the radios that has overridden `check_set_memory_immutable_policy` is not in this whitelist. Perhaps a subclass of one of these? I'll pull it down and see if I can figure it out.

Reply:
It's this one: `BTECH_UV-2501`

I'll modify the error message so it shows the radio in the future.

Reply:
Okay, it's a bunch I guess:
```
        'BTECH_UV-2501',
        'BTECH_UV-2501+220',
        'BTECH_UV-5001',
        'WACCOM_MINI-8900',
        'QYT_KT-UV980',
        'QYT_KT8900',
        'QYT_KT8900R',
        'LUITON_LT-588UV',
        'BTECH_UV-25X2',
        'BTECH_UV-25X4',
        'BTECH_UV-50X2',
```

So we probably need to do something to separate those out, because I'm assuming they all don't need the same override rules and having them in the whitelist like that defeats the point.

Reply:
Okay yeah, so I think what you want is to add the `BTECH_GMRS-50X1` to this whitelist and then only put the `check_memory_set_immutable_policy()` override on the GMRS-20V2, GMRS-50X1, and DB25-G classes, as (from the look of it) only those in `btech.py` are the ones that should have the special behavior, is that right?